src/cluster.c

```diff
@@ -5446,6 +5521,11 @@ void addNodeDetailsToShardReply(client *c, clusterNode *node) {
         addReplyBulkCBuffer(c, node->hostname, sdslen(node->hostname));
         reply_count++;
     }
+    if (sdslen(node->human_nodename) != 0) {
+        addReplyBulkCString(c, "nodename");
+        addReplyBulkCBuffer(c, node->human_nodename, sdslen(node->human_nodename));
+        reply_count++;
+    }
```

Comment:
I'm going to stick behind we probably don't want this in the SHARDS output. There was another discussion where we should probably have an admin variant of shards: https://github.com/redis/redis/issues/11354. I think this would be appropriate there.

Reply:
Sure  , I have removed the nodename from shards output.